scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseRepository.java

```diff
@@ -25,7 +25,7 @@
    * @param cluster couchbase cluster
    * @param couchbaseAdmin couchbase operations with admin permissions
    */
-  public CouchbaseDataAccess(
+  public CouchbaseRepository(
```

Comment:
repository instance seems to be coupled with the name of repository, and could be the field of the object

Reply:
i didnt really understand the comment can you please elaborate?

Reply:
just saw that we are passing this repositoryname here and there and seems it's always the same ... nevermind, i see that doesn't concern the pr actually